app/controllers/events_controller.rb

```diff
@@ -0,0 +1,74 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: [:show, :edit, :update, :destroy]
```

Comment:
I'd really like to learn more about controllers, this looks great and Id like to give you more detailed comments in the future. Can we pick this apart line for line at our next meeting? :)

Reply:
I can definitely help you with this, it is under the Callbacks section of Rails Guides and we will look at it together too!

For a brief explanation: Before running any of the actions in the list (:show, :edit, :update, :destroy), it runs the set_event method down at the bottom of the file. This just dries up the code so we don't have to write set_event in every single action (a convenience method).